src/locales/ca/translation.json

```diff
@@ -0,0 +1,186 @@
+{
+  "home": {
+    "featured": "Destacats",
+    "recently_added": "Afegits recentment",
+    "trending": "Populars",
+    "surprise_me": "Sorprèn-me",
+    "no_results": "No s'ha trobat res"
+  },
+  "sidebar": {
+    "catalogue": "Catàleg",
+    "downloads": "Baixades",
+    "settings": "Configuració",
+    "my_library": "Biblioteca",
+    "downloading_metadata": "{{title}} (S'estan baixant les metadades…)",
+    "checking_files": "{{title}} ({{percentage}} - S'estan comprovant els fitxers…)",
+    "paused": "{{title}} (Pausat)",
+    "downloading": "{{title}} ({{percentage}} - S'està baixant…)",
+    "filter": "Filtra la biblioteca",
+    "follow_us": "Segueix-nos",
+    "home": "Inici",
+    "discord": "Uneix-te al nostre Discord",
+    "telegram": "Uneix-te al nostre Telegram",
+    "x": "Segueix-nos al X",
+    "github": "Col·labora al GitHub"
```

Comment:
These social network strings are not used anymore (we haven't removed them in the english file in the main branch yet), you can also remove them

Reply:
```suggestion
```